validate-schema.py

```diff
@@ -0,0 +1,39 @@
+"""
+A simple script to validate all series YAMLs.
+
+This does not enforce `format` as this is auto-off in JSON Schema's spec.
+
+Note that this does some monkey-patching to dodge around features of PyYAML.
+"""
+import logging
+from pathlib import Path
+from jsonschema import FormatChecker
+from jsonschema.validators import extend, validate, Draft202012Validator
+import yaml, json
+
+# Bodge to ignore dates and just use as-is.
+# This makes this script unsafe to import!
+yaml.SafeLoader.yaml_implicit_resolvers = {
+    k: [r for r in v if r[0] != 'tag:yaml.org,2002:timestamp'] for
+    k, v in yaml.SafeLoader.yaml_implicit_resolvers.items()
+}
+
+Validator = extend(Draft202012Validator, format_checker=FormatChecker())
+
+if __name__ == "__main__":
+    errored = False
+    with open("src/schemata/series.schema.json") as f:
+        schema = json.load(f)
+    
+    series = {}
+    for filename in Path("src/series/").glob("*"):
+        with open(filename, encoding="utf-8") as f:
+            series[filename] = yaml.safe_load(f)
+            try:
+                validate(series[filename], schema, cls=Validator)
+            except Exception as e:
+                logging.error(e, exc_info=True)
```

Comment:
`exc_info=True` makes this pretty verbose for more casual submitters? also it doesn't always print the filename when outputting errors, like if I change genre to RPGaaaa

```suggestion
                print('ERROR in', filename.name)
                logging.error(e)
```